tests/network/vmi_multus.go

```diff
@@ -1362,7 +1362,8 @@ var _ = SIGDescribe("Macvtap", func() {
 			var serverVMIPodName string
 			var serverIP string
 
-			macvtapIfaceIPReportTimeout := 4 * time.Minute
+			// Consider here the default domain resync period (300 sec / 5 min) plus a couple of extra minutes
+			macvtapIfaceIPReportTimeout := 7 * time.Minute
```

Comment:
In fact this should be fixed by https://github.com/kubevirt/kubevirt/pull/6997 . The NIC address info is repoted pretty quickly by guest agent. The problem is that it takes a while to propagate the update to VMI status. The delay happens because in virt-handler the doman update events are currently ignored. The mentioned PR addresses that issue. When its merged this change will not be needed. But frankly speaking I do not have good explanation why this was not observed with Fedora 32.

Reply:
I understood that the ref PR is not closed, but generally speaking, whenever a timer is increased it feels like we are masking a real problem. We have many other tests that are waiting for the guest-agent to report back to the VMI status, so changing it only here is too odd.

I would feel much better if this will not be needed under this change, but fix it in a pre-change. Said that, I also do not feel comftable in blocking this on an unrelated potential bug.

Reply:
> whenever a timer is increased it feels like we are masking a real problem

I fully agree here. I got some feedback on that PR and seems that the proposed solution was not the right one. I could clearly observe that the guest got the address for the secondary NIC rather quickly and then GA reported it. But the update of the vmi status was taking too long. I will need to debug it further.

Reply:
This is fixed by https://github.com/kubevirt/kubevirt/pull/7131